pricing-page-exercise/onder/styles.css

```diff
@@ -0,0 +1,215 @@
+* {
+    box-sizing: border-box;
+}
+
+html {
+    font-family: "Open Sans", sans-serif;
+    font-size: 16px;
+}
+body {
+    margin: 0;
+    padding: 0;
+  background-color: rgb(220, 220, 220);
+}
+/* navbar section */
+
+.navbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1.25rem;
+    background: rgb(180, 180, 180);
+    height: 70px;
+    border-bottom: 1px solid rgb(222 226 230);
+    box-shadow: rgba(0, 0, 0, 0.05) 0px 7px 11px 0px;  
+}
+
+.heading-small {
+    font-size: 1.2rem;
+    font-weight: 400;
+    text-align: left;
+}
+
+.navbar ul {
+    display: flex;
+    align-items: center;
+    gap: 1.2rem; 
+    list-style-type: none;
+}
+
+.navlink {
+    text-decoration: none;
+    color: #363636;
+    font-size: 1rem;
+}
+.navlink:hover {
+    text-decoration:underline;
+    color:#363636;
+}
+
+.text-center {
+    text-align: center;
+  font-weight: bold;
+}
+
+
+.btn {
+    color:rgb(30, 143, 218);
```

Comment:
Try to only use one kind of colors in your project (usually either HEX or RGB is used, however HEX is more common). It's ok to mix HEX and RGBA which you need to provide the "A" value for opacity.